bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MyStromConfiguration.java

```diff
@@ -12,24 +12,77 @@
  */
 package org.openhab.binding.mystrom.internal;
 
-import static org.openhab.binding.mystrom.internal.MyStromBindingConstants.DEFAULT_REFRESH_RATE_SECONDS;
+import static org.openhab.binding.mystrom.internal.MyStromBindingConstants.*;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 
 /**
  * The {@link MyStromConfiguration} class contains fields mapping thing configuration parameters.
  *
  * @author Paul Frank - Initial contribution
+ * @author Stefan Navratil - Added configuration for myStrom PIR
  */
 @NonNullByDefault
 public class MyStromConfiguration {
 
+    private final String url_prefix = "http://";
```

Comment:
According to naming convention:
```suggestion
    private final String urlPrefix = "http://";
```